### PR TITLE
Add option to get `figure_path` from environment variables

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
### Overview

This PR adds the option to get `FIGURE_PATH` global `pyvista` variable from the environment variables of the OS.

### CODE REVIEW

**Comment by MatthewFlamm** [comment]
Will need #5196 , but otherwise LGTM

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Comment by AlejandroFernandezLuces** [comment]
Let's wait for the other PR to be merged then. Thank you for your reviews 🙂

**Pull request merged**
